src/Microsoft.Health.Fhir.Core/Features/Search/Expressions/IExpressionVisitor.cs

```diff
@@ -74,5 +74,12 @@ public interface IExpressionVisitor<in TContext, out TOutput>
         /// <param name="expression">The expression to visit.</param>
         /// <param name="context">The input.</param>
         TOutput VisitInclude(IncludeExpression expression, TContext context);
+
+        /// <summary>
+        /// Visits the <see cref="SortParameterExpression"/>.
+        /// </summary>
+        /// <param name="expression">The expression to visit.</param>
+        /// <param name="context">The input</param>
+        TOutput VisitSortParameter(SortParameterExpression expression, TContext context);
```

Comment:
SortParameterExpression -> SortExpression ?
I see that IncludeExpression is not IncludeParameterExpression

Reply:
I dont see the change. still see SortParam...